app/styles/partials/nav-bar.scss

```diff
@@ -0,0 +1,4 @@
+.right.menu > .item > .ui.blue.button {
```

Comment:
@anu-007 Please use something like 
```
.nav-bar > .item > .ui.blue.button {
}
```
Add this `nav-bar` class to the div containing `right menu` class.